CalculatorExercises/1-CalculatorBasic/MainTest.java

```diff
@@ -0,0 +1,27 @@
+import java.util.*;
+import java.lang.*;
+
+public class MainTest {
```

Comment:
rename to Main, MainTest sounds like class that is testing other class named Main